.circleci/config.yml

```diff
@@ -127,31 +185,181 @@ jobs:
             python3 --version
             pip3 --version
       - run:
-          name: Setup Tests / Scanner Requirements
+          name: Load images
           command: |
-            make test_setup
+            docker load -i workspace_cache/splunk-debian-10.tar
+            docker load -i workspace_cache/uf-debian-10.tar
       - run:
-          name: Build Container
-          command: |
-            make all
+          name: Setup
+          command: make test_setup
       - run:
-          name: Export Build Images for Artifacts
-          command: |
-            make save_containers
-      - run:
-          name: Test Python3 installation
-          command: make test_python3_all
-      - run:
-          name: Test Python2 as the default
-          command: make test_python2_all
+          name: Run large tests
+          command: make run_large_tests_debian10
+          no_output_timeout: 1h
       - store_artifacts:
           path: test-results
           destination: test-results
+      - store_test_results:
+          path: test-results
+
 workflows:
   version: 2
-  build:
+  pipeline:
     jobs:
-      - security-scanning
-      - debian10-testing
-      - container-validation
-      - redhat8-testing
\ No newline at end of file
+      - build_debian_10
```

Comment:
Is there any parallelism here?  Should we run these distros in parallel to try and speed up the CI?

Reply:
Yeah, they already are: https://app.circleci.com/pipelines/github/splunk/docker-splunk/938/workflows/aaf37ec1-d0b3-49e9-8f65-eac16cb260fa